index.js

```diff
@@ -50,17 +51,22 @@ const lookupHandler = function (params) {
             return;
         }
 
+        let userAlreadyInQueue = queue.isUserInQueue(params.ip);
+
         // If the flag is set, check if the user already has a request in the queue
         if (!CONFIG.allow_simultaneous_requests && userAlreadyInQueue) {
             resHandler.respondErrorToUser(params, {error: errorMsgs[3], code: 3}, 400);
             return;
         }
 
-        queue.addJob(params, CONFIG.bot_settings.max_attempts, () => {
-            if (params.type === 'ws') {
-                resHandler.respondInfoToUser(params, {'msg': `Your request for ${params.a} is in the queue`});
-            }
-        });
+        queue.addJob(params, CONFIG.bot_settings.max_attempts);
+
+        if (params.type === 'ws') {
+            resHandler.respondInfoToUser(params, {'msg': `Your request for ${params.a} is in the queue`});
+        }
+    })
+    .catch((err) => {
+        resHandler.respondErrorToUser(params, {error: `${errorMsgs[6]}: ${err.message}`, code: 6}, 500);
```

Comment:
I don't think we should give the user a detailed error message. I doubt they would care to report it nor understand what it is returning. Also, with a detailed error, this message would be much longer than any of the other errors (for UIs and such).